queries/haskell/highlights.scm

```diff
@@ -4,66 +4,33 @@
 ; and don't override destructured parameters
 (variable) @variable
 
-(pat_wildcard) @variable
+(pattern/wildcard) @variable
 
-(function
-  patterns: (patterns
-    (_) @variable.parameter))
+(decl/function patterns: (patterns
+  (_) @variable.parameter))
 
-(exp_lambda
-  (_)+ @variable.parameter
-  "->")
+(expression/lambda (_)+ @variable.parameter
```

Comment:
Is it missing a run or is it already done? This looks a bit off.

Reply:
nvm the formatter goes even worse

Reply:
Freaking hell. Why does half the time the result is correct and the other half it's completely wrong ??????????

Reply:
supertypes! \o/

Reply:
oof. Sorry :cry:

Reply:
Nah, that's our problem; supertypes are official, and we'll have to support them sooner or later. We'll get there.